tests/jwt_vc.rs

```diff
@@ -36,11 +40,14 @@ pub async fn wallet_verifier() -> (JwtVcWallet, Arc<Verifier>) {
         )
         .unwrap(),
     );
+
+    let resolver = VerificationMethodDIDResolver::new(DIDJWK);
+
     let client = Arc::new(
-        oid4vp::verifier::client::DIDClient::new(
+        oid4vp::verifier::client::DIDClient::new::<AnyMethod>(
             verifier_did_vm.clone(),
             signer.clone(),
-            DIDKey.to_resolver(),
+            &resolver,
```

Comment:
I changed the `DIDClient` constructor to take `VerificationMethodDIDResolver`. Not sure if this is the correct verification method resolver or setup.